.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -218,6 +218,7 @@ steps:
       -e DRY_RUN=${DRY_RUN} \
       -e IMAGE_NAME=${IMAGE_NAME} \
       -e VNET_RESOURCE_GROUP_NAME=${VNET_RESOURCE_GROUP_NAME} \
+      -e TEST_VM_RESOURCE_GROUP_NAME=${TEST_VM_RESOURCE_GROUP_NAME} \
```

Comment:
How is`TEST_VM_RESOURCE_GROUP_NAME` generated in `vhdbuilder/packer/test/run-test.sh` passed here?